api-inference-community/setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name="api_inference_community",
-    version="0.0.21",
+    version="0.0.23",
```

Comment:
Shouldn't it be `0.0.22` ?

Reply:
No it's 0.0.23 Since it's 0.0 I publish a bit faster than necessary :)